.github/workflows/yarn-plugin-integration-test.yml

```diff
@@ -0,0 +1,58 @@
+
+name: integration test of yarn-plugin
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Check repository out
+        uses: actions/checkout@v4
+        with:
+            repository: opossum-tool/OpossumUI
+            path: OpossumUI
+
+      - name: install
+        run: |
+            set -e
+            cd OpossumUI
+            yarn install
+            yarn plugin import https://raw.githubusercontent.com/spdx/tools-ts/main/yarn-plugin/bundles/@yarnpkg/plugin-spdx.js
+
+      - name: test yarn-plugin help message
+        run: |
+            set -e
+            cd OpossumUI
+            yarn spdx --help
+
+      - name: test yarn-plugin
+        run: |
+            set -e
+            cd OpossumUI
+            yarn spdx
+            [[ -f opossum-ui.spdx.json ]]
+            set -x
+            wc opossum-ui.spdx.json
```

Comment:
These are only intended for visual inspection of the CI results, right?
I don't see a case where this would fail.

Reply:
yes, this just sumarisses